src/Nethermind/Nethermind.Blockchain/ITracer.cs

```diff
@@ -30,6 +30,7 @@ public interface ITracer
         GethLikeTxTrace Trace(long blockNumber, Transaction transaction, GethTraceOptions options);
         GethLikeTxTrace Trace(long blockNumber, int txIndex, GethTraceOptions options);
         GethLikeTxTrace Trace(Keccak blockHash, int txIndex, GethTraceOptions options);
+        GethLikeTxTrace Trace(Rlp blockRlp, Keccak txHash, GethTraceOptions options);
```

Comment:
please change to txindex so it is consistent with other methods

Reply:
but its the one with hash not index

Reply:
is this header supported by other clients? then fine should stay the same

Reply:
I don't think so, but its for our debugging purposes. I am not sure why we are considering other clients